optimizely/odp/odp_event_manager.py

```diff
@@ -64,7 +67,14 @@ def __init__(
 
         self.event_queue: Queue[OdpEvent | Signal] = Queue(OdpEventManagerConfig.DEFAULT_QUEUE_CAPACITY)
         self.batch_size = OdpEventManagerConfig.DEFAULT_BATCH_SIZE
-        self.flush_interval = OdpEventManagerConfig.DEFAULT_FLUSH_INTERVAL
+
+        if flush_interval is None:
+            self.flush_interval = OdpEventManagerConfig.DEFAULT_FLUSH_INTERVAL
+        elif flush_interval == 0:
+            self.flush_interval = 0
+        else:
+            self.flush_interval = flush_interval
```

Comment:
How about we condense this block down to:
```suggestion
        self.flush_interval = OdpEventManagerConfig.DEFAULT_FLUSH_INTERVAL if flush_interval is None else flush_interval
```